src/db/sqlite/lrsql/sqlite/sql/query.sql

```diff
@@ -327,3 +327,10 @@ WHERE id > :since-id
   AND id <= :until-id
 GROUP BY stored_time
 ORDER BY stored_time ASC;
+
+-- :name query-statement-ids-by-actor
+-- :command :query
+-- :result :many
+SELECT x.id FROM xapi_statement x
+LEFT JOIN statement_to_actor sta ON x.statement_id=sta.statement_id
```

Comment:
this is for sure not getting what you need.

Reply:
Oh wait i see it a bit more clearly now. So this is to get the initial set then you'll have the delete fns weed out the dependencies.